contracts/src/bridge/Bridge.sol

```diff
@@ -230,8 +230,11 @@ contract Bridge is Initializable, DelegateCallAware, IBridge {
         InOutInfo storage info = allowedDelayedInboxesMap[inbox];
         bool alreadyEnabled = info.allowed;
         emit InboxToggle(inbox, enabled);
-        if ((alreadyEnabled && enabled) || (!alreadyEnabled && !enabled)) {
-            return;
+        /// @solidity memory-safe-assembly
+        assembly {
+            if eq(alreadyEnabled, enabled) {
```

Comment:
I think we should use the `==` solution instead of the assembly. These functions are called very infrequently so I value simplicity in them over any gas optimisation